CHANGELOG.md

```diff
@@ -13,6 +13,13 @@
     * Playbooks
     * Integrations
 * Improved user outputs for **secrets** command.
+* Added auto-complete functionality to demisto-sdk.
+* Added git parameter in **lint** to run only on changed packages.
+* Added the **run-playbook** command
+* Added **run** command which runs a command in the Demisto playground.
+* Added **upload** command which uploads an integration or a script to a Demisto instance.
+* Fixed and issue where **validate** checked if release notes exist for new integrations and scripts.
+
```

Comment:
Added **json-to-outputs** command which generates the yaml section for outputs from an API raw response.